code/modules/events/fake_virus.dm

```diff
@@ -1,6 +1,8 @@
 /datum/event/fake_virus/start()
 	var/list/fake_virus_victims = list()
 	for(var/mob/living/carbon/human/victim in shuffle(GLOB.player_list))
+		if(ismachineperson(victim))
+			continue
```

Comment:
why not just check for the virus immunity trait?

Reply:
Done